x-pack/plugins/observability_solution/entity_manager/server/templates/components/base_history.ts

```diff
@@ -7,8 +7,8 @@
 
 import { ClusterPutComponentTemplateRequest } from '@elastic/elasticsearch/lib/api/types';
 
-export const entitiesBaseComponentTemplateConfig: ClusterPutComponentTemplateRequest = {
-  name: 'entities_v1_base',
+export const entitiesHistoryBaseComponentTemplateConfig: ClusterPutComponentTemplateRequest = {
```

Comment:
whilst we're looking at these with a critical eye - do we need `labels` and `tags` in these templates? tags will be handled by the dynamic metadata mappings, and we don't support objects in the output (yet)

Reply:
I don't think it hurts anything... I guess we could remove the base from the latest.